sal_interfaces/MTAlignment/MTAlignment_Commands.xml

```diff
@@ -81,4 +145,92 @@
       <Count>1</Count>
     </item>
   </SALCommand>
+  <SALCommand>
+    <Subsystem>MTAlignment</Subsystem>
+    <EFDB_Topic>MTAlignment_command_setReferenceGroup</EFDB_Topic>
+    <item>
+      <EFDB_Name>referenceGroup</EFDB_Name>
+      <Description>Nominal point group to locate tracker station to and provide data relative to</Description>
```

Comment:
Please use the same style for all descriptions. This one starts with an upper case letter while all others start with a lower case letter. I'd prefer you change all others to start with an upper case letter (because that is what descriptions in ts_xml generally look like) but will accept if you change this one to lower case.